fake-java.equivs

```diff
@@ -0,0 +1,5 @@
+Package: java17-runtime
+Version: 17
+Architecture: all
+Description: dummy java17-runtime package
+ A dummy package to avoid dpkg dependency issues.
```

Comment:
Nasqueron code follows an inclusive language policy.

The `dummy` word can be associated to "negative perceptions of neurodivergent people and people unable to speak".

https://www.incits.org/dotAsset/29643f45-86d8-4137-987e-9685b944d1e0.pdf suggests "placeholder" or "mock" as an alternative.